### ISSUE TITLE
[Feature Request] 1.19.2 Compatibility

### ISSUE THREAD

**Issue by ghost** [lilypuree/Decorative-blocks-compat#2]
I'd like to request an update to 1.19.2. I love this mod and would like to use it in a Fabric pack in conjunction with custom recipes to implement the creative items into the game.

**Comment by WolfsRise** [comment]
YES PLEASE! This mod looks so cool and I'd love to add it to my personal pack! (Forge update)

**Comment by IceflowRE** [comment]
every-compat is the mod you search.
